R/server.R

```diff
@@ -813,6 +973,7 @@ shinyServer(function(input, output, session) {
         },
         content = function(file) {
             yaml::write_yaml(list(coursewide = policy$coursewide,
+                                  slip_days = unname(policy$slip_days),
```

Comment:
What's `unnames()` for?

Reply:
because it was creating an extra line in the yaml file -- so it was messing up the structure. I could not figure out how else to make this work.